app/views/idv/shared/_verify.html.erb

```diff
@@ -50,7 +56,7 @@ locals:
         </div>
         <div>
           <dt class="display-inline"> <%= t('idv.form.state') %>: </dt>
-          <dd class="display-inline margin-left-0"> <%= pii[:state_id_jurisdiction] %> </dd>
+          <dd class="display-inline margin-left-0"> <%= pii[:state_id_state] %> </dd>
```

Comment:
If you expand the code above this, which was collapsed by github, you can see that this is the portion of the code which displays the state id address.  Since the state id address could include a different state than the issuing state, I believe this needs to be `state_id_state`.